lib/ansible/modules/windows/win_route.ps1

```diff
@@ -0,0 +1,171 @@
+#!powershell
+# This file is part of Ansible
+#
+# Copyright 2016, Daniele Lazzari <lazzari@mailup.com>
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# WANT_JSON
+# POWERSHELL_COMMON
+
+# win_route (Add or remove a network static route)
+
+$params = Parse-Args $args;
+
+$destip = Get-Attr $params "destination_ip" -failifempty $true;
```

Comment:
We now use Get-AnsibleParam and -type for each parameter. (See for example win_file or win_stat)
Trailing semi-colon is not needed in modules.